pkg/datastore/noinstancer_test.go

```diff
@@ -42,60 +45,63 @@ func ExampleDataStore_noInstancer() {
 		SERVICE_ADMIN:   dbrole.WRITER,
 	}
 
-	if err := ds.Register(DevInstanceCtx, roleMapping, &xUser{}); err != nil {
+	if err := ds.Register(DevInstanceCtx, roleMapping, p1); err != nil {
```

Comment:
We always pass empty structs to `Register` method, because the contents of the struct are not used at all